### PR TITLE
[doc]: fix s2e-env virtual environment commands

### PR DESCRIPTION
**Pull request by dingiso**
the commands in `s2e-env.rst` have a little mistakes. this pr will fix them.

### CODE REVIEW

File: docs/src/s2e-env.rst [hunk 1]
```diff
@@ -32,7 +32,8 @@ Installing s2e-env
     cd s2e-env
 
     python3 -m venv venv
-    . venv/bin/activate
+    chmod +x ./venv/bin/activate 
+    ./venv/bin/activate
```
Comment (vitalych) [inline docs/src/s2e-env.rst:36]: The command was correct. This script must be sourced.
https://docs.python.org/3/tutorial/venv.html

Comment (dingiso) [reply docs/src/s2e-env.rst:36]: Thanks V. Using `source` is actually what I have done. I'm consider to change it to `source` just right now

**Review by vitalych — APPROVED** [review]

**Review by vitalych — CHANGES_REQUESTED** [review]
Please squash commits and rebase.

**Comment by dingiso** [comment]
> Please squash commits and rebase.

I have rebased it with one single commit

**Pull request closed without merging**